src/types/query.rs

```diff
@@ -10,7 +10,7 @@ use std::convert::TryFrom;
 /// * `order_by` is responsible for defining the order in which the response will be represented, **optional**. Argument is a vector with strings containing the element and how to order (`:asc` or `:desc`) `vec!["time :desc", "device-id :asc"]`.
 /// * `limit` is responsible for defining the limit size of the response, **optional**. Argument is a usize.
 /// * `offset` is responsible for defining the offset of the response, **optional**. Argument is a usize.
-#[derive(Clone)]
+#[derive(Clone, Debug)]
 pub struct Query {
```

Comment:
Added Debug for testing purposes

Reply:
But are you going to remove it?

Reply:
Or is it a permanent thing to help the crate users?

Reply:
When using with ATM-Crux I noticed that this reduced testability.
What do you think?